test/gtest/common/test_helpers.h

```diff
@@ -12,13 +12,18 @@
 
 #include <ucs/sys/preprocessor.h>
 #include <ucs/sys/checker.h>
+#include <ifaddrs.h>
+#include <net/if_arp.h>
```

Comment:
why need all those includes in test_helpers.h? can move to test_helpers.cc?

Reply:
only #include <netinet/in.h> is needed for sockaddr which is also used in the header file. the rest are leftovers - i'll remove.

Reply:
actually, struct sockaddr is defined in sys/socket.h